tests/unit/test_grid.py

```diff
@@ -21,19 +21,24 @@
     compute_horizontal_source_grids,
     compute_num_elements,
     compute_projected_horizontal_source_grids,
-    compute_source_swath,
     compute_target_area,
+    create_area_definition_for_source_grid,
+    create_target_area_from_source,
     crs_from_source_data,
     dims_are_lon_lat,
     dims_are_projected_x_y,
+    get_area_definition_from_message,
     grid_height,
     grid_width,
 )
 
 
-@patch('harmony_regridding_service.grid.AreaDefinition', wraps=AreaDefinition)
-def test_compute_target_area(mock_area):
+@patch('harmony_regridding_service.grid.get_area_definition_from_message')
+def test_compute_target_area_with_parameters(
+    mock_get_area_definition_from_message, smap_projected_netcdf_file, var_info_fxn
+):
```

Comment:
This test isn't great.  you're setting up a mock to make sure it's calling the correct internal function, which is good, but then the assertions are just looking at what you set the mock to.  I'm not sure there's value there, I will return to this tomorrow and give you some advice.